modules/ROOT/pages/migration/index.adoc

```diff
@@ -24,17 +25,48 @@ Here is a list of all the breaking changes from version 6.0.0 to 7.0.0.
 
 The `@unique` directive is no longer supported:
 
-
 [source, graphql, indent=0]
 ----
 type Movie {
-    title: String! @unique()
+    title: String! @unique
 }
 ----
```

Comment:
Is the type definition example useful do you think?

Reply:
Actually not!